src/Framework/BuildSubmissionStartedEventArgs.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Framework
+{
+    [Serializable]
```

Comment:
`[Serializable]` is not needed for new events (or old ones for that matter, we should go back and clean 'em up right @JanKrivanek?)

Reply:
Absolutely.
They'll eventually became obsolete and we'll be forced to do so anyways